### DISCUSSION TITLE
Option to open session/console in a floating window

### DISCUSSION THREAD

**Discussion by wurli** [posit-dev/positron#4284] [Feedback]
This would be really nice for cases when the traditional split screen doesn't offer enough real-estate, e.g. for viewing large tibbles etc. Personally I like to have my terminal open on another monitor entirely - VSCode/Positron already support this as shown, but unfortunately not yet with the console!

<img width="297" alt="image" src="https://github.com/user-attachments/assets/af4350a2-038a-4c75-8411-8dfbc3d72bca">

Loving Positron so far, thanks for all the work to create such an awesome free product!

**Comment by juliasilge** [comment]
Thanks for this suggestion! 🙌 I'll make an issue from this discussion and we can look into how it can be prioritized.

**Reply by juliasilge** [reply]
You can track it here, if you like: https://github.com/posit-dev/positron/issues/4294
